Description: Install missing files as MANIFEST.in is wrong if using git-buildpackage
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed

--- ceilometer-2013.1~g0.3+ad5ff3e4e5.orig/MANIFEST.in
+++ ceilometer-2013.1~g0.3+ad5ff3e4e5/MANIFEST.in
@@ -6,3 +6,14 @@ exclude .gitreview
 
 global-exclude *.pyc
 recursive-include public *
+include ceilometer/api/v1/static/bootstrap/css/bootstrap-responsive.css
+include ceilometer/api/v1/static/bootstrap/css/bootstrap.css
+include ceilometer/api/v1/static/bootstrap/img/glyphicons-halflings-white.png
+include ceilometer/api/v1/static/bootstrap/img/glyphicons-halflings.png
+include ceilometer/api/v1/static/bootstrap/js/bootstrap.js
+include ceilometer/api/v1/static/d3.v2.js
+include ceilometer/api/v1/static/rickshaw.css
+include ceilometer/api/v1/static/rickshaw.js
+include ceilometer/api/v1/templates/list_event.html
+include ceilometer/storage/sqlalchemy/migrate_repo/README
+include ceilometer/storage/sqlalchemy/migrate_repo/migrate.cfg
